scripts/install_karavi_auth.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Done. I've added to install_karavi_auth.sh and karavi-authorization.spec. Didn't add to yaml file since other yaml files didn't have it and this one didn't have it before either. Please let me know if it's necessary for that as well